openedx/features/badging/tests/test_helpers.py

```diff
@@ -105,9 +106,8 @@ def test_get_course_badges_with_empty_badge_queryset(self, mock_badge_objects_al
     @mock.patch('openedx.features.badging.helpers.badges.is_teams_feature_enabled')
     @mock.patch('openedx.features.badging.helpers.badges.get_course_by_id')
     def test_get_course_badges_successfully(self, mock_get_course_by_id,
-                                        mock_is_teams_feature_enabled,
-                                        mock_filter_earned_badge_by_joined_team,
-                                        mock_add_badge_earned_date):  # pylint: disable=unused-argument
+            mock_is_teams_feature_enabled, mock_filter_earned_badge_by_joined_team,
+            mock_add_badge_earned_date):  # pylint: disable=unused-argument
```

Comment:
It's still not according to [edx recommendations](https://edx.readthedocs.io/projects/edx-developer-guide/en/latest/style_guides/python-guidelines.html#breaking-long-lines). It should be like:

`    def test_get_course_badges_successfully(
        self,
        mock_get_course_by_id,
        mock_is_teams_feature_enabled,
        mock_filter_earned_badge_by_joined_team,
        mock_add_badge_earned_date  # pylint: disable=unused-argument`
    ):

Please see if statement to disable pylint warning is correctly placed on the above code snippet.

Please update accordingly in lines 168-169 as well.

Reply:
The link you are referring to is for calling the function not to define a function. Please go through [pep8](https://www.python.org/dev/peps/pep-0008/) 
```
# Add 4 spaces (an extra level of indentation) to distinguish arguments from the rest.
```

Reply:
Ok, then lets please follow the edx recommendation. Right now the arguments are over intended (there should be 1 extra level of indentation before arguments according to what you shared. Right now there are about 13 extra levels of indentation).

Please update on both places.

Reply:
Please see the examples in pep8 and compare it with my code. I hope it will be helpful to understand. 

here is example code
```
# Add 4 spaces (an extra level of indentation) to distinguish arguments from the rest.
def long_function_name(
        var_one, var_two, var_three,
        var_four):
    print(var_one)
```

There is an extra level of indentation from method body. That's what I am following.